teams/teams-ps/teams/New-CsTeamsMediaConnectivityPolicy.md

```diff
@@ -0,0 +1,68 @@
+---
+Module Name: microsoftteams
+applicable: Microsoft Teams
+title: New-CsTeamsMediaConnectivityPolicy
+online version: https://learn.microsoft.com/powershell/module/teams/New-CsTeamsMediaConnectivityPolicy
+schema: 2.0.0
+author: lirunping_MSFT
+ms.author: runli
+---
+
+
+# New-CsTeamsMediaConnectivityPolicy
+
+## SYNOPSIS
+This cmdlet creates a Teams media connectivity policy.
+
+## SYNTAX
+
+```powershell
+New-CsTeamsMediaConnectivityPolicy -Identity <String>
+```
+
+## DESCRIPTION
+This cmdlet creates a Teams media connectivity policy. If you get an error that the policy already exists, it means that the policy already exists for your tenant. In this case, run Get-CsTeamsMediaConnectivityPolicy.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> New-CsTeamsMediaConnectivityPolicy -Identity Test
+
+Identity DirectConnection
+-------------------------
+Tag:Test Enabled
+```
+
+Returns the current value of "DirectConnection" policy with identity of "Test".
```

Comment:
It looks like the cmdlet "creates" instead of "returns"...

Reply:
Yes. It firstly create then returns(actually prints) the newly created policy. I'll update to make it more clear.